datacatalog/cloud-client/src/main/java/com/example/datacatalog/LookupEntryBigQueryDataset.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.datacatalog;
+
+import com.google.cloud.datacatalog.Entry;
+import com.google.cloud.datacatalog.LookupEntryRequest;
+import com.google.cloud.datacatalog.v1beta1.DataCatalogClient;
+
+public class LookupEntryBigQueryDataset {
+
+  /**
+   * Lookup the Data Catalog entry referring to a BigQuery Dataset
+   *
+   * @param projectId The project ID to which the Dataset belongs, e.g. 'my-project'
+   * @param datasetId The dataset ID to which the Catalog Entry refers, e.g. 'my_dataset'
+   * @param lookupBySqlResource Indicates whether the lookup should be performed by Sql Resource
+   *     instead of Linked Resource, e.g. 'false'
+   */
+  private static void lookupEntryBigQueryDataset(
+      String projectId, String datasetId, boolean lookupBySqlResource) {
+
+    LookupEntryRequest request;
+
+    // Construct the Lookup request to be sent by the client.
+    if (lookupBySqlResource) {
```

Comment:
Please add some additional context around these statements. What is the difference between these two things? Which one should I use in which scenarios? Are these easy to find id's associated with the resource, or is there information available on how I should use them?

Reply:
They are alternatives to do the same thing. I commented out the sqlResource part. Please let me know if it works.